piksi_tools/serial_link.py

```diff
@@ -258,6 +263,16 @@ def run(args, link):
     # SystemExit exception.
     sys.exit(1)
 
+# The default whitelist of messages for the Piksi to publish
+DEFAULT_WHITELIST = [SBP_MSG_HEARTBEAT,
```

Comment:
I think we want more messages than this, right? (e.g., uart data for latencies, etc., etc.) Should we either ignore this or move this to a blacklist?